modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -45,16 +45,12 @@ endif::[]
 |Cloned Disk
 |Provision source is a cloned disk.
 
-|Import
-|Import virtual machine from a supported provider.
-
 |Operating System
-|
-|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine. If you select `Import` as the *Provider Source*, the operating system is filled in automatically, based on the operating system of the VMware virtual machine being imported.
+|A list of operating systems available in the cluster. This is the primary operating system for the virtual machine.
 
 |Flavor
 |small, medium, large, tiny, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine.
+|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for Flavor are determined by the operating system and Workload profile.
```

Comment:
We might want to remove "Workload profile" from this b/c it comes after the workload profile. This is something that will be adjusted for the next release.

Reply:
Added the updated sentence for presets.
Removing Workload profile from this location.

Reply:
Matt - please confirm - do we want to remove Workload Profile and the three options: desktop, generic, high performance - all of this info from this location?

Reply:
We want to keep workload profile I just think we can drop it from the sentence so it could say " The presets displayed for flavor are determined by the Operating System selection."